src/main/java/com/cms/claimmanagement/insuranceCompany/claim/service/ClaimService.java

```diff
@@ -17,6 +17,9 @@
 
 @Service
 public class ClaimService {
+    private final String claimPrefix = "CL";
```

Comment:
If it is a constant, maybe it could be static. 
If it is static, it should follow the java convention, for this case it would be CLAIM_PREFIX